LM-1201/99-beta/conditionals.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
Should this be python3? Apparently it helps make sure the system knows what libraries might be there by default.